tutorralph/plugin.py

```diff
@@ -149,29 +173,29 @@
     ],
 )
 
-
 hooks.Filters.ENV_PATCHES.add_item(
     (
         "local-docker-compose-services",
         f"""  
 ralph:
-    #image: fundocker/ralph:master
-    image: ralph:development
+    image: docker.io/fundocker/ralph:master
     depends_on:
       - clickhouse
     env_file:
       - ../../env/plugins/ralph/apps/config/env
     ports:
       - "8100:8100"
     command:
+      - python 
+      - "-m"
       - ralph
       - "-v"
       - DEBUG
       - runserver
       - "-b"
-      - "mongo"
+      - "clickhouse"
     volumes:
-      - /Users/brianmesick/Dev/forks/ralph:/app
+      - ../../env/plugins/ralph/apps/config/.ralph:/app/.ralph
```

Comment:
There does seem to be some kind of issue with Ralph spinning up before Clickhouse in spite of the depends_on, I'll look into that today. If you're seeing the `statements` response it's because the database is empty. My install wasn't loading the test xapi data by default, but I'll turn that on today and make sure it's all working.

Reply:
Ralph reporting no statements on demo data is fixed here: https://github.com/openedx/tutor-contrib-oars/pull/9

Reply:
I can confirm that this works eventually, but it didn't work from a fresh install :( Please see https://github.com/openedx/tutor-contrib-oars/pull/9#discussion_r1134749884